app/Models/Post.php

```diff
@@ -709,10 +701,10 @@ public function getReferralPostEventPayload()
                 'type' => $this->type,
                 'status' => $this->status,
                 'action_id' => $this->action_id,
-                'created_at' => $this->created_at->toIso8601String(),
-                'updated_at' => $this->updated_at->toIso8601String(),
+                'created_at' => $this->created_at->timestamp,
+                'updated_at' => $this->updated_at->timestamp,
```

Comment:
This change is made for consistency with other dates we send to Customer.io. Customer.io can only do "date operations" when given a UNIX timestamp, and so these would otherwise just be treated as plain old strings.